app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -84,7 +86,22 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 shareIntent.putExtra(Intent.EXTRA_TEXT, shareSub)
                 startActivity(Intent.createChooser(shareIntent, getString(R.string.msg_share_using)))
             }
+            resources.getString(R.string.title_rate_us) ->
+                startAppPlayStore()
+        }
+    }
 
+    fun getMarketAppLink(): String {
+        return "market://details?id=chat.rocket.android"
```

Comment:
It'll be better if you put this link in the `defaults.xml` file like you did for the `play_store_link` string there.